src/sql/parts/dataExplorer/objectExplorer/connectionViewlet/connectionViewletPanel.ts

```diff
@@ -111,7 +111,7 @@ export class ConnectionViewletPanel extends ViewletPanel {
 
 	layoutBody(size: number): void {
 		this._searchBox.layout();
-		this._serverTreeView.layout(size); // account for search box
+		this._serverTreeView.layout(size - 46); // account for search box and horizontal scroll bar
```

Comment:
is this going to do the right thing when the horizontal scrollbar isn't visible?

Reply:
Yes, the searchBox is 36px and the scrollBar is 10px.